examples/native-fungible/src/state.rs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+use linera_sdk::views::{linera_views, RegisterView, RootView, ViewStorageContext};
+use thiserror::Error;
+
+/// The application state.
+#[derive(RootView, async_graphql::SimpleObject)]
+// In the service we also want a metadata query, which is not derived from a struct member.
+// This attribute requires having a ComplexObject implementation that adds such fields.
+// The implementation with metadata is in service.rs. Since a ComplexObject impl is required,
+// there is also an empty one in contract.rs.
+#[graphql(complex)]
+#[view(context = "ViewStorageContext")]
+pub struct NativeFungibleToken {
+    // TODO(#968): We should support stateless Applications/empty user views
+    pub _dummy: RegisterView<u8>,
```

Comment:
`RegisterView<()>` ?

Reply:
I think to do that we need to implement graphql's `OutputType` for `()`, which from my understanding I don't think we can do :/ 
That's probably why we also use `u8` in the AMM